wallet-ui-store.html

```diff
@@ -109,13 +109,18 @@ <h6>Wallet Contents:</h6>
     });
 
     document.getElementById('doneButton').addEventListener('click', () => {
-      returnToUser(event, 'success');
+      returnToUser(event, vp);
     });
   }
 
-  function returnToUser(storeEvent, status) {
+  /**
+   * @param storeEvent
+   * @param {VerifiablePresentation|null} data - Return (to client application)
+   *   exactly what was stored, or a `null` if canceled by the user.
+   */
+  function returnToUser(storeEvent, data) {
     storeEvent.respondWith(new Promise(resolve => {
-      return resolve({dataType: 'Response', data: status});
+      return resolve({dataType: 'VerifiablePresentation', data});
```

Comment:
This should resolve to `null` when canceled, not a null VerifiablePresentation. (`resolve(null)`)

Reply:
Ah, ok.

Reply:
Fixed.